Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -41,7 +41,7 @@
         "package": "PanModal",
         "repositoryURL": "https://github.com/brave/PanModal",
         "state": {
-          "branch": "master",
+          "branch": null,
```

Comment:
this ok?

Reply:
Yeah I kept seeing it pop up so I let it through. Its Xcode fixing up its own mistake, it should have `null`'d out the branch when I changed it to a commit, but it didnt for some reason